tests/integration/utils2.go

```diff
@@ -1301,6 +1300,7 @@ func createDocViaGQL(
 	node client.DB,
 	collections []client.Collection,
 ) ([]*client.Document, error) {
+	println(fmt.Sprint(collections))
```

Comment:
question: is it left here intentional?

Reply:
It is not :)  And I should have been more careful so soon after the mutation type bug :)  Thanks for spotting

- [ ] remove print